src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -215,12 +252,60 @@ public View TitleContent
             }
         }
 
+        /// <summary>
+        /// Action views of AppBar.
+        /// Action views are added to ActionContent of AppBar.
+        /// The Action and ActionButton styles of AppBarStyle are applied to actions only by setting Actions.
+        /// If you do not want to apply Action and ActionButton styles to action views, then please use ActionContent.Add() instead.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View[] Actions
```

Comment:
Please check build warning : Properties should not return arrays.

Reply:
Thank you for your comment!
I have removed the getter of Actions.

Reply:
It looks like these two warnings are because Actions have only setter.
Should we suppress the messages of CA1044 and CA1819?

Reply:
Do not use Array on public API

Reply:
Below code will be help to implement it

 https://github.com/xamarin/Xamarin.Forms/blob/caab66bcf9614aca0c0805d560a34e176d196e17/Xamarin.Forms.Core/Page.cs#L130

https://github.com/xamarin/Xamarin.Forms/blob/caab66bcf9614aca0c0805d560a34e176d196e17/Xamarin.Forms.Core/Page.cs#L62-L64

https://github.com/xamarin/Xamarin.Forms/blob/caab66bcf9614aca0c0805d560a34e176d196e17/Xamarin.Forms.Core/Page.cs#L530-L536

Reply:
Normally, ObservableCollection is used to handle adding and removing item by API user

But, it does not expose original type, due to below rule
✔️ DO use the least-specialized type possible as a parameter type. Most members taking collections as parameters use the IEnumerable<T> interface.

Reply:
@rabbitfor @Seoyeon2Kim @myroot 
Could you let me know which one would be better between the followings?

**1. Use Property with ReadOnlyCollection**
```
// Property
public ReadOnlyCollection<View> Actions
{
    get
    {
       ...
    }
    set
    {
        ...
    }
}

// Usage
var page = new ContentPage()
{
    AppBar = new AppBar()
    {
        Actions = new ReadOnlyCollection<View>(new List<view> { new Button() { ... }, ... }),
    },
    ...
};
```

**2. Use Method instead of Property**
```
// Method
public void SetActions(params View[] actions)
{
    ...
}

// Usage
var page = new ContentPage()
{
    ...
};

var appBar = new AppBar();
appBar.SetActions(new Button() { }, ...);

page.AppBar = appBar;
```

Reply:
I prefer readonly property, because method can't use on XAML directly

for example 
``` xml
   <AppBar>
      <AppBar.Actions>
          <Button Text="....">
          <Button Text="....">
     </AppBar.Actions>
  </AppBar>
```

Reply:
But I don't know that you guys are willing to support XAML and MVVM(data binding)

Reply:
Thank you for your comment.
The code has been updated.